pl_examples/intel_compressor/config/quantization.yaml

```diff
@@ -0,0 +1,20 @@
+version: 1.0
+
+model:
+  name: imagenet
+  framework: pytorch_fx                              # supported values are pytorch, pytorch_fx, pytorch_ipex...and so on.
+
+device: cpu
+
+quantization:
+  approach: post_training_static_quant               # supported values are post_training_static_quant, post_training_dynamic_quant, quant_aware_training.
+  calibration:
+    sampling_size: 300
+
+tuning:
+  accuracy_criterion:
```

Comment:
This is quite confusing. My metric might not be accuracy and yet it is used as the decisive criterion to stop tuning. Might be worth to reconsider a better terminoligy.

Reply:
@tchaton , How about "metric_criterion" or "metric"?